web-registry/src/lib/db/types/json-ld/project-metadata.ts

```diff
@@ -1,60 +1,70 @@
-import { TypeValue, UrlList, UrlType } from 'lib/rdf/types';
+import { NameUrl, TypeValue, UrlList, UrlType } from 'lib/rdf/types';
 
+/** Anchored metadata AKA "Additional Info" - Editable only with a signed Ledger TX. */
 export interface ProjectMetadataLD {
   '@context': Context;
-  '@type': string;
+  '@type': string; // ex: regen:C01-Project
   'schema:name': string;
-  'schema:image': UrlType;
-  'schema:creditText': string;
-  'schema:description'?: string;
   'schema:location': any;
   'regen:projectType': string;
-  'regen:projectActivity': ProjectActivity;
-  'regen:projectDeveloper': ProjectStakeholder;
-  'regen:landSteward': ProjectStakeholder;
-  'regen:landOwner': ProjectStakeholder;
-  'regen:projectOriginator': ProjectStakeholder;
+  'regen:projectActivity': NameUrl;
   'regen:offsetGenerationMethod': string;
   // TODO: not sure why regen:offsetGenerationMethod isn't compacting as expected so added this:
   'http://regen.network/offsetGenerationMethod'?: string;
   'regen:projectSize': ProjectSize;
   'regen:projectStartDate': TypeValue;
   'regen:projectEndDate': TypeValue;
-  'regen:boundaries': TypeValue;
-  'regen:creditClass': CreditClass;
-  'regen:landManagementActions': LandManagementActions;
-  'regen:galleryPhotos': UrlList;
-  'regen:previewPhoto': UrlType;
-  'regen:videoURL': UrlType;
-  'regen:glanceText': any;
-  'regen:landStory': string;
-  'regen:landStewardStory': string;
-  'regen:landStewardPhoto': UrlType;
-  'regen:projectQuote': ProjectQuote;
-  'regen:landStewardStoryTitle': string;
+  'regen:projectDeveloper': ProjectStakeholder;
+  'regen:landSteward': ProjectStakeholder;
+  'regen:landOwner': ProjectStakeholder;
+  'regen:projectOriginator': ProjectStakeholder;
+}
+
+/** Un-anchored metadata from our DB. This is editable without a Ledger TX. */
+export interface ProjectPageMetadataLD {
+  '@context': Context;
+  '@type': string; // regen:Project-Page
+  'regen:creditClassId': string;
+
+  'regen:previewPhoto'?: UrlType;
+  'schema:image'?: UrlType; // exist?
+  'schema:creditText'?: string;
+  'regen:galleryPhotos'?: UrlList;
+  'regen:videoURL'?: UrlType;
+  'regen:glanceText'?: any;
+  'regen:landStory'?: string;
+  'regen:landStewardStory'?: string;
+  'regen:landStewardStoryTitle'?: string;
+  'regen:landStewardPhoto'?: UrlType;
+  'regen:projectQuote'?: ProjectQuote;
+  'schema:description'?: string;
+  'regen:boundaries'?: TypeValue;
+  'regen:creditClass'?: CreditClass;
+  'regen:landManagementActions'?: LandManagementActions;
+
+  // Optional reference IDs:
```

Comment:
@blushi Discussing with @wgwz , these may be helpful knowlege graph reference IDs. If we decide to keep them here in `ProjectPageMetadataLD`, we will need to make sure they get updated at the same time as the anchored ones.